src/GitHub.UI/Controls/ComboBoxes/LinkDropDown.cs

```diff
@@ -87,9 +87,18 @@ private void UpdateLinkText()
                 // will work as long as DisplayMemberPath is just a property name, which is
                 // all we need right now.
                 if (string.IsNullOrWhiteSpace(DisplayMemberPath))
-                {
-                    LinkText = item.ToString();
+                {                   
+                    if (ItemTemplate == null)
```

Comment:
If I'm reading this correctly, then yes, we can't have `DisplayMemberPath` and `ItemTemplate` set at the same time. So if  for some reason either of those are not set then we can just display the item.